playwright/test/search-images.test.ts

```diff
@@ -9,7 +9,7 @@ import {
   testIfFilterIsApplied,
 } from './helpers/search';
 
-export const ItemViewerURLRegex = /\/works\/[a-zA-Z0-9]+\/images[?]id=/;
+const ItemViewerURLRegex = /\/works\/[a-zA-Z0-9]+\/images[?]id=/;
```

Comment:
Just curious why the export didn't work?

Reply:
Huh ok they don't allow it interesting! The regex could maybe be moved to https://github.com/wellcomecollection/wellcomecollection.org/blob/main/playwright/test/helpers/utils.ts  ?